marshalable-derive/src/lib.rs

```diff
@@ -1,14 +1,28 @@
 #![forbid(unsafe_code)]
 
-use std::collections::HashMap;
-
 use proc_macro2::{Span, TokenStream};
 use quote::{quote, quote_spanned};
+use std::collections::HashMap;
 use syn::{
     parse_macro_input, spanned::Spanned, Attribute, Data, DataEnum, DeriveInput, Error, Expr,
-    Field, Fields, FieldsNamed, Ident, Index, MetaNameValue, Path, Result, Type,
+    ExprCall, ExprPath, Field, Fields, FieldsNamed, Ident, Index, MetaNameValue, Path,
+    PathArguments, Result, Type,
 };
 
+// Enable for logging during compilation. Remember to call "init_logger();" at initialization.
```

Comment:
to be removed ?

Reply:
It was used for dumping syntax tree at compile time, to support during development. Removed again.